### PR TITLE
Add IsBetween<>

### PR DESCRIPTION
**Pull request by fdarosa2663**

### CODE REVIEW

**Comment by fdarosa2663** [comment]
just pinging this PR to see if any changes were needed

File: functions/isbetween.h [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+template<class F, class BOTTOM, class TOP>
+class IsBetweenBase {
+public:
+  void run(BladeBase* blade) {
+    f_.run(blade);
+    bottom_.run(blade);
+    top_.run(blade);
+  }
+  int getInteger(int led) {
+    if (f_.getInteger(led) > bottom_.getInteger(led) && f_.getInteger(led) < top_.getInteger(led)) {
```
Comment (profezzorn) [inline functions/isbetween.h:20]: We need to use a temporary variable instead of calling f_.getInteger(led) twice.

File: functions/isbetween.h [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  PONUA TOP top_;
+};
+
+template<class SVFA, class SVFB, class SVFC>
+class IsBetweenSVF {
+ public:
+  void run(BladeBase* blade) {
+    svfa_.run(blade);
+    svfb_.run(blade);
+    svfb_.run(blade);
```
Comment (profezzorn) [inline functions/isbetween.h:39]: should probably be svfc_

File: functions/isbetween.h [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  }
+  int calculate(BladeBase* blade) {
+    if (svfa_.calculate(blade) > svfb_.calculate(blade) && svfa_.calculate(blade) < svfc_.calculate(blade)) {
+      return 32768;
+    } else {
+      return 0;
+    }
+  }
+ private:
+  PONUA SVFA svfa_;
```
Comment (profezzorn) [inline functions/isbetween.h:49]: these need better names, like svf_f_, svf_top_ and svf_bottom_

File: functions/isbetween.h (hunk 1 above)
Comment (fdarosa2663) [reply functions/isbetween.h]: added

File: functions/isbetween.h (hunk 2 above)
Comment (fdarosa2663) [reply functions/isbetween.h]: renamed and fixed

File: functions/isbetween.h (hunk 3 above)
Comment (fdarosa2663) [reply functions/isbetween.h]: renamed

**Pull request merged**